druid/src/contexts.rs

```diff
@@ -238,7 +238,7 @@ impl<'a> EventCtx<'a> {
         self.is_handled
     }
 
-    /// The focus status of a widget.
+    /// The (tree) focus status of a widget.
```

Comment:
Maybe also something like,

```suggestion
    /// The (tree) focus status of a widget.
    ///
    /// This is used principally for event routing; it is `true` iff
    /// this widget or one of its descendants may have focus.
```